modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/ObjectFolderExportImportTest.java

```diff
@@ -0,0 +1,264 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.web.internal.portlet.action.test;
+
+import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+import com.liferay.object.admin.rest.dto.v1_0.ObjectFolder;
+import com.liferay.object.admin.rest.dto.v1_0.ObjectFolderItem;
+import com.liferay.object.admin.rest.resource.v1_0.ObjectFolderResource;
+import com.liferay.object.constants.ObjectFolderConstants;
+import com.liferay.petra.function.transform.TransformUtil;
+import com.liferay.portal.kernel.json.JSONFactory;
+import com.liferay.portal.kernel.portlet.bridges.mvc.MVCActionCommand;
+import com.liferay.portal.kernel.portlet.bridges.mvc.MVCResourceCommand;
+import com.liferay.portal.kernel.test.util.TestPropsValues;
+import com.liferay.portal.kernel.util.ListUtil;
+import com.liferay.portal.test.rule.FeatureFlags;
+import com.liferay.portal.test.rule.Inject;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+import com.liferay.portal.vulcan.pagination.Page;
+import com.liferay.portal.vulcan.pagination.Pagination;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Guilherme Sa
+ */
+@FeatureFlags("LPS-148856")
+@RunWith(Arquillian.class)
+public class ObjectFolderExportImportTest extends BaseExportImportTestCase {
+
+	@ClassRule
+	@Rule
+	public static final LiferayIntegrationTestRule liferayIntegrationTestRule =
+		new LiferayIntegrationTestRule();
+
+	@Before
+	public void setUp() throws Exception {
+		user = TestPropsValues.getUser();
+
+		ObjectFolderResource.Builder builder =
+			_objectFolderResourceFactory.create();
+
+		_objectFolderResource = builder.user(
+			user
+		).build();
+	}
+
+	@Test
+	public void testExportImportObjectFolder() throws Exception {
+		testExportImport(
+			"empty_object_folder.json", "EmptyObjectFolderERC",
+			"EmptyObjectFolder");
+
+		_assertObjectFolder(
+			"EmptyObjectFolder", 0, Collections.emptyList(),
+			Collections.emptyList());
+
+		testExportImport(
+			"object_folder_invalid_items.json", "empty_object_folder.json",
+			"EmptyObjectFolderERC", "EmptyObjectFolder");
+
+		_assertObjectFolder(
+			"EmptyObjectFolder", 0, Collections.emptyList(),
+			Collections.emptyList());
+
+		//  Import and export a linked object folder item that doesn't exist
```

Comment:
I think a better message would be `Import and export a folder with a linked object folder item that doesn't exist`. Its more verbose, but currently its sounding like we are importing and exporting a single object folder item